server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/GetMailboxesMethod.java

```diff
@@ -121,31 +125,43 @@ private GetMailboxesResponse getMailboxesResponse(GetMailboxesRequest mailboxesR
             .orElseGet(Throwing.supplier(() -> retrieveAllMailboxes(mailboxSession)).sneakyThrow());
     }
 
+
+
     private Stream<Mailbox> retrieveSpecificMailboxes(MailboxSession mailboxSession, ImmutableList<MailboxId> mailboxIds) {
         return mailboxIds
             .stream()
-            .map(mailboxId -> mailboxFactory.builder()
+            .map(mailboxId ->  mailboxFactory.builder()
```

Comment:
you should run a checkstyle on the commits

Reply:
that extra space seems to be fixed in the commit after this one